lib/utilities/extensions.dart

```diff
@@ -137,4 +141,23 @@ extension StringExt on String {
   }
 
   String get toBaseExp => '$this xp';
+
+  String get shortcut => replaceAll('Special', 'Sp')
+      .replaceAll('attack', 'Atk')
+      .replaceAll('defense', 'Def');
+
+  Color get baseStatsColor {
+    var color;
+    var value = double.parse(this) / 200;
+
+    if (value < 0.25) {
+      color = Color(0xfffa6e70);
```

Comment:
Please create a color.dart file 

Here is an example what color.dart file will contain
`const Color white = Color(0xFFFFFFFF);`

then you will only call that color into this line 
color = white